test/client.test.ts

```diff
@@ -1687,19 +1690,21 @@ if (getEdgeDBVersion().major >= 2) {
 
 if (!isDeno && getAvailableFeatures().has("admin-ui")) {
   test("binary protocol over http", async () => {
+    const tokenFile = path.join(__dirname, "keys", "jwt");
+    const token = fs.readFileSync(tokenFile);
```

Comment:
This file is used in the deno tests, so I think it will break if you import 'fs' or 'path' at the top level. Instead I think you need to import from `src/adapter.node.ts` or use `require` inside the test:
```suggestion
    const tokenFile = require('path').join(__dirname, "keys", "jwt");
    const token = require('fs').readFileSync(tokenFile);
```